[CP-148] Added new gRPC API, and make OpenAPI deprecated. (#1681)
qdrant/landing_page: qdrant-landing/content/documentation/cloud-api.md

aliases:
  - /documentation/qdrant-cloud-api/
---
# Qdrant Cloud API: Powerful gRPC and Flexible REST/JSON Interfaces

> **Note:** This is not the Qdrant REST or gPRC API of the database itself. For core product APIs & SDKs, see our list of [interfaces](/documentation/interfaces/)

Reviewer: ```suggestion
**Note:** This is not the Qdrant REST or gPRC API of the database itself. For database APIs & SDKs, see our list of [interfaces](/documentation/interfaces/)
```

`core` is something more internal, others may not understand what this means.

## Introduction

> The Qdrant Cloud API lets you manage Cloud accounts and their respective Qdrant clusters. You can use this API to manage your clusters, backup schedules, authentication methods, hybrid cloud environments, and and many more.

Reviewer: ```suggestion
The Qdrant Cloud API lets you automate the Qdrant Cloud platform. You can use this API to manage your accounts, clusters, backup schedules, authentication methods, hybrid cloud environments, and more.
```

You can find the API definitions and generated client libraries in our Qdrant Cloud Public API [GitHub repository](https://github.com/qdrant/qdrant-cloud-public-api).
**Note:** The API is splitted into multiple services to make it easier to use.

### Qdrant Cloud API Endpoints

* **gRPC Endpoint**: grpc.cloud.qdrant.io:443
* **REST/JSON Endpoint**: https://api.cloud.qdrant.io

### Authentication

Most of the Qdrant Cloud API requests must be authenticated. Authentication is handled via API keys (so called management keys), which should be passed in the Authorization header.
**Management Keys**: `Authorization: apikey <YOUR_MANAGEMENT_KEY>`

Replace <YOUR_MANAGEMENT_KEY> with the actual API key obtained from your Qdrant Cloud dashboard or generated programmatically.

You can create a Cloud API key in the Cloud Console UI. Go to **Access Management** > **Qdrant Cloud API Keys**.
![Authentication](/documentation/cloud/authentication.png)

**Note:** Ensure that the API key is kept secure and not exposed in public repositories or logs.  Once authenticated, the API allows you to manage clusters, backup schedules, and perform other operations available to your account.

### Samples

> For samples how to use the API, with a tool (like grpcurl, curl or any of the provided SDKs), please see the [Qdrant Cloud Public API](https://github.com/qdrant/qdrant-cloud-public-api) repository.

Reviewer: ```suggestion
For samples on how to use the API, with a tool like grpcurl, curl or any of the provided SDKs, please see the [Qdrant Cloud Public API](https://github.com/qdrant/qdrant-cloud-public-api) repository.
```
